### ISSUE TITLE
Unable to Compile

### ISSUE THREAD

**Issue by H3LD3R** [delafer/SkrE3Dip_Ender5#1]
Hi!
i'm having issues compiling... do you have a newer version?

SKR DIP e3 with 4x2208 on a Ender 5

Thanks in advance

**Comment by delafer** [comment]
yes, i have... but you can use official latest version from branch and use/replace only your config files. Which issues do you have during compilation?

**Comment by SergejM88** [comment]
Hallo
Habe ein SKR DIP e3 mit 4x2208 auf einem Ender 5.habe im konfigurationsdatei auf tmc2208 geändert und danach kompiliert erfolgreich.aber wenn ich sd card mit dem bin datei rein stecke es piept dauernd und passiert nichts mehr

**Comment by roel111** [comment]
hey I am upgrading my ender 5 to SKR DIP e3 with 4x2208 I want to load the program but I get an error message

Marlin \ src \ HAL \ STM32F1 \ ../../ inc /../ HAL /./ STM32F1 / fastio.h: 30: 10: fatal error: libmaple / gpio.h: No such file or directory

can you help me already thanks
